src/cdn/beer.ts

```diff
@@ -65,27 +66,29 @@ export default (() => {
     element.removeEventListener(name, callback, true);
   }
 
-  function insertBefore (newElement: Element, element: Element): Element {
-    return element?.parentNode.insertBefore(newElement, element);
+  function insertBefore (newElement: Element, element: Element): Element | undefined {
+    return element?.parentNode?.insertBefore(newElement, element);
   }
 
-  function prev (element: Element): Element {
+  function prev (element: Element): Element | null {
     return element?.previousElementSibling;
   }
 
-  function next (element: Element): Element {
+  function next (element: Element): Element | null {
     return element?.nextElementSibling;
   }
 
-  function parent (element: Element): Element {
+  function parent (element: Element): Element | null {
     return element?.parentElement;
   }
 
-  function create (json: any): HTMLElement {
+  function create (json: { [key: string]: string | null }): HTMLElement {
```

Comment:
I don't understand this params. There are no type to represent that we need here?